Sources/ApolloCodegenLib/Frontend/GraphQLSchema.swift

```diff
@@ -75,7 +75,7 @@ public class GraphQLInputField: JavaScriptObject {
   
   private(set) lazy var description: String? = self["description"]
   
-  lazy var defaultValue: Any? = self["defaultValue"]
+  lazy var defaultValue: GraphQLValue? = (self["astNode"] as JavaScriptObject)["defaultValue"]
```

Comment:
🥇

Reply:
Haha thank you. This was painful to figure out... The other solution was going to require a lot of additional code in the TypeScript library that was going to further slow down the compilation of the AST. This is a little hacky, but definitely a better solution.

Long term, I'd really like us to re-write the `JavascriptFrontend` in Swift and just access the cold hard JavaScript objects in Swift directly from the start. It will be ugly and unsafe code, casting the JS objects directly without the TypeScript compiler checking and converting types for us, but it should improve performance further. Not urgent at all, but possibly a cool intern project or something haha. :)